src/util/constants.template.ts

```diff
@@ -25,7 +25,7 @@ export let PRICING_PERIOD = BigInt.fromI32(21600);  // 6 hours
 export let USDT_WETH_PAIR = '{{usdt_weth_address}}';
 export let UNISWAP_FACTORY = '{{uniswap_factory}}';
 export let SUSHI_FACTORY = '{{sushi_factory}}';
-export let MIN_USD_PRICING = BigDecimal.fromString('10000.0');
+export let MIN_USD_PRICING = BigDecimal.fromString('1000.0');
```

Comment:
We've been keeping mainnet at 10,000 and polygon at 1,500. Are you thinking we just make both of them 1000? I'm cool with it if thats the case, just wanted to see what you were thinking

Reply:
Nope, just wanted to move polygon to 1000. If we merge this in, will change it back